coralproject/talk
# only close modal if click originates from baseScroll

## What does this PR do?

This PR addresses https://vmproduct.atlassian.net/browse/CORL-2258 and https://vmproduct.atlassian.net/browse/CORL-2255 by determining which click events originate from children of the baseScroll component of the modal as opposed to the baseScroll it's self.

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## How do I test this PR?
Open any modal, verify that scrolling works (if needed), verify that click some element inside the modal does not cause the modal to close, and verify that clicking outside the modal does close it.

## How do we deploy this PR?
There shouldn't be any special considerations for deploying this fix.
